### DISCUSSION TITLE
Font Ligatures

### DISCUSSION THREAD

**Discussion by benjcal** [orbitalquark/textadept#333] [Q&A] [answered]
Is there a way to disable font ligatures in ta apart from using a font without ligatures? I'm using `Fira Code` and if it was possible I'd still like to keep using it but without ligatures. 

Thanks :)

**Comment by orbitalquark** [comment, marked as answer]
Sorry, I don’t know anything about ligatures and don’t handling :(

**Comment by benjcal** [comment]
just for future reference, I simply switched to another font for the time being

**Comment by paaguti** [comment]
For future reference... I'd make this customisable. Some of us are very
happy with the current behaviour ;-)
/PA

On Wed, 11 Jan 2023 at 02:44, Ben Calderon ***@***.***> wrote:

> just for future reference, I simply switched to another font for the time
> being
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orbitalquark/textadept/discussions/333#discussioncomment-4651517>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAZKU4AAQ6YEXSQOGGMSIP3WRYF7ZANCNFSM6AAAAAATXMZSZE>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***
> com>
>


-- 
Fragen sind nicht da um beantwortet zu werden,
Fragen sind da um gestellt zu werden
Georg Kreisler

Headaches with a Juju log:
unit-basic-16: 09:17:36 WARNING juju.worker.uniter.operation we should run
a leader-deposed hook here, but we can't yet

**Comment by rgieseke** [comment]
For further future reference:

Scintilla's discussions on ligatures: https://groups.google.com/g/scintilla-interest/search?q=ligature

For Fira code here someone discussed building a Fira version without fl ligatures: https://github.com/tonsky/FiraCode/issues/849
(not sure if successful).
